src/lib/pipeline.ml

```diff
@@ -392,6 +402,8 @@ let rec to_file_prefix:
       sprintf "%s-%s-%s" prev
         vc.Variant_caller.name
         vc.Variant_caller.configuration_name
+    | Rna_hla_typer s ->
+      sprintf "RNA_HLA_typer-%s" (to_file_prefix ?read s)
```

Comment:
This should be specific to the name of the typer (in this case, Seq2HLA); otherwise navigating nodes can't tell which typer is being run